### PR TITLE
Merge v2 to upstream as of 09/20/2021

### PR DESCRIPTION
**Pull request by nearora-msft**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: .github/workflows/e2e_multi_az_test.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: Run e2e multi-az tests
```
Comment (edreed) [inline .github/workflows/e2e_multi_az_test.yaml:1]: This file needs to be removed upstream.

**Review by andyzhangx — APPROVED** [review]
/approve

**Pull request merged**